ui/lib/core/addon/components/object-list-input.hbs

```diff
@@ -0,0 +1,47 @@
+{{yield}}
+{{#each this.inputList as |row index|}}
+  <div class="field is-grouped">
+    {{#each-in row as |inputKey value|}}
+      {{#let (find-by "key" inputKey @objectKeys) as |field|}}
+        <div class="control is-expanded">
+          {{#if (eq index 0)}}
+            <h2 data-test-object-list-label={{inputKey}}>{{field.label}}</h2>
+          {{/if}}
+          <Input
+            data-test-object-list-input="{{field.key}}-{{index}}"
+            aria-label={{if field.placeholder "{{field.placeholder}} for {{index}}" "{{field.label}} for {{index}}"}}
+            class="input"
+            placeholder={{or field.placeholder ""}}
+            @type="text"
+            @value={{value}}
+            name={{field.key}}
+            id="{{field.key}}-{{index}}"
+            {{on "input" (fn this.handleInput index)}}
+          />
+        </div>
+      {{/let}}
+    {{/each-in}}
+    <div class="control is-align-end">
+      {{#if (eq (inc index) this.inputList.length)}}
+        <button
+          data-test-object-list-add-button
+          type="button"
+          class="button is-outlined is-primary"
+          disabled={{this.disableAdd}}
+          {{on "click" this.addRow}}
+        >
+          Add
+        </button>
+      {{else}}
+        <button
+          data-test-object-list-delete-button={{index}}
+          type="button"
+          class="button is-icon"
+          {{on "click" (fn this.removeRow index)}}
```

Comment:
can we add an aria-label for non-sighted friends? 
```suggestion
          aria-label="remove"
          {{on "click" (fn this.removeRow index)}}
```

Reply:
ah yes! thank you!